### PR TITLE
Remove 1st character in URL if name starts with @

### LINKED ISSUE #4: Links to non-existent URL when @ is used for username
Clicking on the text selected by the freehand square leads me to:

### PR DESCRIPTION
**Pull request by Unihedro**
fixes #4

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -69,7 +69,7 @@
             clearChildren(username);
             var a = document.createElement("a");
             a.setAttribute("target", "blank");
-            a.href = "https://github.com/" + user;
+            a.href = "https://github.com/" + (user.startsWith("@") ? user.slice(1) : user);
```
Comment (IonicaBizau) [inline js/main.js]: Could you please use a regular expression instead? Something like `user.replace(/^\@/, "")`.

Comment (Unihedro) [inline js/main.js]: Why? This is a trivial task that can be done with a simple native method, and involves literal strings - nothing that we can't do without regexes. I hate to leave you off with a rant, but using regexes for tasks like these will result in poor performance and is definitely why we can't have nice things: http://h2co3.org/blog/index.php/2014/07/28/this-is-why-we-cant-have-nice-things/

Comment (IonicaBizau) [inline js/main.js]: I see.

However, then please use `user.charAt(0) === "@"` instead of using `startsWith` which is an experimental thing.

Comment (Unihedro) [inline js/main.js]: I agree with that.

**Comment by IonicaBizau** [comment]
@Unihedro Please add yourself as contributor [here](https://github.com/IonicaBizau/github-profile-languages/blob/f45ed23617edddbdb8b6707a76f764e518e5c37d/package.json#L20). :cake:

**Comment by Unihedro** [comment]
Ok, thanks for the credits ;D

**Comment by IonicaBizau** [comment]
Thanks a lot! Merged in `1.2.0`. :ship: :fireworks: :star2:

**Pull request merged**